#
# Remove for possible confusion about support website
#
--- libdvbtee-orig/libdvbtee/parse.cpp	2013-01-13 21:30:14.022439253 -0500
+++ libdvbtee/libdvbtee/parse.cpp	2013-01-13 21:31:19.801848742 -0500
@@ -576,8 +576,8 @@
   , rewritten_pat_cont_ctr(0)
 {
 	if (!hello)
-		fprintf(stderr, "dvbtee v" LIBDVBTEE_VERSION ", built " __DATE__ " " __TIME__
-			"\nMichael Krufky - http://github.com/mkrufky\n\n");
+		fprintf(stdout, "dvbtee v" LIBDVBTEE_VERSION ", built " __DATE__ " " __TIME__
+			"\n\thttp://github.com/mkrufky/libdvbtee\n");
 	hello = true;
 	dprintf("()");
 
